pkg/cmd/cli/topic/config_topic.go

```diff
@@ -1 +1,70 @@
 package topic
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/electric-saw/kafta/internal/pkg/configuration"
+	"github.com/electric-saw/kafta/internal/pkg/kafka"
+	cmdutil "github.com/electric-saw/kafta/pkg/cmd/util"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+type listConfigsOptions struct {
+	config *configuration.Configuration
+	topic  string
+}
+
+func NewCmdListConfigs(config *configuration.Configuration) *cobra.Command {
+	options := &listConfigsOptions{config: config}
+	cmd := &cobra.Command{
+		Use:   "list-configs TOPIC",
+		Short: "List all configurations for a topic",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			options.topic = args[0]
+			cmdutil.CheckErr(options.run())
+		},
+	}
+
+	return cmd
+}
+
+func (o *listConfigsOptions) run() error {
+	conn := kafka.MakeConnection(o.config)
+	defer conn.Close()
+
+	resource := sarama.ConfigResource{
+		Name: o.topic,
+		Type: sarama.TopicResource,
+	}
+	configs, err := conn.Admin.DescribeConfig(resource)
+	if err != nil {
+		return fmt.Errorf("failed to describe config for topic %s: %w", o.topic, err)
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
```

Comment:
you can use `utils.PrintTable` to show the output \o/

Reply:
Done !! I added a slice organizer, but I don't know if it will be very useful, if you want I can remove it ! thaks for review